jenkinsapi/build.py

```diff
@@ -93,12 +93,17 @@ def get_params(self):
         Return a dictionary of params names and their values or None
         if no parameters present
         """
+        # This is what a parameter action looks like:
+        # {'_class': 'hudson.model.ParametersAction', 'parameters': [
+        #     {'_class': 'hudson.model.StringParameterValue',
+        #      'value': '12',
+        #      'name': 'FOO_BAR_BAZ'}]}
         actions = self._data.get('actions')
         if actions:
             parameters = {}
             for elem in actions:
-                if 'parameters' in elem:
-                    parameters = elem['parameters']
+                if elem.get('_class') == 'hudson.model.ParametersAction':
```

Comment:
I can't see the reason for using `dict.get()` if default value is not provided and/or key is not optional.